packages/mantine/src/components/table/Table.tsx

```diff
@@ -61,9 +49,13 @@ export const Table: TableType = <T,>({
 
     const {predicates, dateRange, ...initialStateWithoutForm} = initialState;
     const form = useForm<TableFormType>({
-        initialValues: {predicates: initialState?.predicates ?? {}, dateRange: initialState?.dateRange ?? [null, null]},
+        initialValues: {
+            predicates: initialState?.predicates ?? {},
+            dateRange: initialState?.dateRange ?? [null, null],
+            layout: layouts[0].name,
```

Comment:
Just wondering if we should let the user set the layout in the initialState

Reply:
Enabled it, also added a UT for it.